lib/mix/phoenix/scope.ex

```diff
@@ -25,6 +27,7 @@ defmodule Mix.Phoenix.Scope do
       scope
       | name: name,
         alias: alias,
+        route_access_path: scope.route_access_path || scope.access_path,
```

Comment:
I am thinking it should default to either `Enum.drop(scope.access_path, -1)` or simply `[name]`. Thoughts? I really don't think we should be generated .id or .slug by default.

Reply:
`Enum.drop(scope.access_path, -1)` is good 👍

Reply:
I guess it could be empty in some cases, then maybe we fallback to scope.access_path as a whole. Unsure.